test/e2e/helm/cluster_test.go

```diff
@@ -64,7 +64,7 @@ var _ = Describe("Running Helm projects", func() {
 		It("should run correctly in a cluster", func() {
 			By("checking if the Operator project Pod is running")
 			verifyControllerUp := func() error {
-				By("getting the controller-manager pod name")
```

Comment:
The By() is helpful on the logs.
So, the initial idea was instead of explaining with // golang comments follow up the framework approach by using the By which indeed will be added in the log and help us to troubleshoot the failure scenarios.

Reply:
The commands being run are already printed so the `By()` calls are unnecessarily verbose.